tests/apps/courses/test_admin_licence.py

```diff
@@ -0,0 +1,82 @@
+"""
+Test suite defining the admin pages for the Licence model
+"""
+from django.urls import reverse
+
+from cms.test_utils.testcases import CMSTestCase
+from parler.utils.context import switch_language
+
+from richie.apps.core.factories import UserFactory
+from richie.apps.courses.factories import LicenceFactory
+
+
+class LicenceAdminTestCase(CMSTestCase):
+    """
+    Integration test suite to validate the behavior of admin pages for the Licence model
+    """
+
+    def test_admin_licence_list_view(self):
+        """
+        The admin list view of licences should display the name of the licence.
+        """
+        user = UserFactory(is_staff=True, is_superuser=True)
+        self.client.login(username=user.username, password="password")
+
+        # Create a licence
+        licence = LicenceFactory()
+
+        # Get the admin list view
+        url = reverse("admin:courses_licence_changelist")
+        response = self.client.get(url, follow=True)
+
+        # Check that the page includes the name field
+        self.assertContains(response, licence.name, status_code=200)
```

Comment:
if you create a licence in several languages you can check that it diplays in the current language.